templates/complete/main.tf

```diff
@@ -69,23 +69,25 @@ module "hubnetworking" {
 }
 
 module "virtual_network_gateway" {
-  source  = "Azure/vnet-gateway/azurerm"
-  version = "0.1.2"
+  source  = "Azure/avm-ptn-vnetgateway/azurerm"
+  version = "0.2.0"
 
   for_each = local.module_virtual_network_gateway
 
   location                            = each.value.location
   name                                = each.value.name
   sku                                 = each.value.sku
-  subnet_address_prefix               = each.value.subnet_address_prefix
   type                                = each.value.type
   virtual_network_name                = each.value.virtual_network_name
   virtual_network_resource_group_name = each.value.virtual_network_resource_group_name
   default_tags                        = try(each.value.default_tags, null)
   edge_zone                           = try(each.value.edge_zone, null)
+  enable_telemetry                    = false
   express_route_circuits              = try(each.value.express_route_circuits, null)
   ip_configurations                   = try(each.value.ip_configurations, null)
   local_network_gateways              = try(each.value.local_network_gateways, null)
+  subnet_address_prefix               = try(each.value.subnet_address_prefix, null)
+  subnet_id                           = try(each.value.subnet_id, null)
```

Comment:
How come `subnet_id` wasn't in the last module? Isn't this an important setting? Does this change any behaviour?

Reply:
No behaviour change, just a new variable that was in a latter release.